packages/trpc/server/routers/viewer/attributes/assignUserToAttribute.handler.ts

```diff
@@ -148,21 +148,9 @@ const assignUserToAttributeHandler = async ({ input, ctx }: GetOptions) => {
           id: true,
         },
       });
-    } else if (!attribute.value && attribute.options && attribute.options.length > 0) {
+    } else if (!attribute.value && attribute.options) {
       const options = attribute.options;
 
-      // Get all users attributes for this attribute
-      await prisma.attributeToUser.findMany({
-        where: {
-          attributeOption: {
-            attribute: {
-              id: attribute.id,
-            },
-          },
-          memberId: membership.id,
-        },
-      });
```

Comment:
Am I right that this code isn't doing anything?

Reply:
This is a good point. I don't think it is